### ISSUE TITLE
[FormattedString] The short syntax for Formatted String not working 

### ISSUE THREAD

**Issue by NickIliev** [NativeScript/NativeScript#4368]
Based on [this documentation article](https://docs.nativescript.org/cookbook/formatted-string-ng#how-to-use-the-formattedstring-class-in-text), we are supporting both short and full syntax in NativeScript core application and recommending using the short one.
However with tns-core-modules 3.0.1 only the full syntax is working.
Test project [here](https://github.com/NickIliev/NativeScript-Issues-2017/tree/master/others/formattedstringissue) - reproducible with both Android and iOS.

**Non-working** short syntax
```
<Label backgroundColor="lightblue">
    <FormattedString>
        <Span text="&#xf0d7; " fontFamily="FontAwesome" />
        <Span text="Test 1234" fontStyle="italic"  />
        <Span text="Test 5678" />
    </FormattedString>
</Label>
```
Note: this seems to be problematic only when theformatted string is placed within `<drawer:RadSideDrawer.mainContent>`

Working:
```
<Label>
    <Label.formattedText>
        <FormattedString>
            <FormattedString.spans>
                <Span text="&#xf0d7; " fontFamily="FontAwesome" />
                <Span text="Test 1234" fontStyle="italic"  />
                <Span text="Test 5678" />
            </FormattedString.spans>
        </FormattedString>
    </Label.formattedText>
</Label>
```

t.1114079

**Comment by sitefinitysteve** [comment]
This is my label code

**Comment by NickIliev** [comment]
@sitefinitysteve even with your exact snippet I am not reproducing this with tns-core-modules 3.0.1

[Here is the test project ](https://github.com/NickIliev/NativeScript-Issues-2017/tree/master/NativeScript/issue_4233_nonNG)I have used - let me know if there are another specifics that are related to this issue (e.g. setting global styles for Label or similar).

The documentation issue is resolved via [https://github.com/NativeScript/docs/pull/810](https://github.com/NativeScript/docs/pull/810)

**Comment by erkanarslan** [comment]
This is still a problem. Not working.

**Comment by NickIliev** [comment]
Resolved in tns-core-modules@4.1.0 - both short and long syntaxes are supported for `FormattedString`

Test application can be found [here](https://github.com/NickIliev/NS-Issues-2018-II/tree/master/NativeScript/issue-4368)
